package.json

```diff
@@ -156,6 +156,27 @@
       "type": "bool",
       "value": true,
       "hidden": true
+    },
+    {
+      "name": "showSearch",
+      "title": "Show the Search bar on the New Tab page",
+      "type": "bool",
+      "hidden": true,
```

Comment:
yeah, I guess so. although it won't make a difference once this is a system addon since we wont be listed in `about:addons` (i think?)